be/src/runtime/user_function_cache.cpp

```diff
@@ -425,43 +425,4 @@ Status UserFunctionCache::get_jarpath(int64_t fid, const std::string& url,
     return Status::OK();
 }
 
-Status UserFunctionCache::check_jar(int64_t fid, const std::string& url,
```

Comment:
why del this code of `check_jar`?

Reply:
now could use get_jarpath function to check this